torchgen/gen_aoti_c_shim.py

```diff
@@ -244,6 +244,7 @@ def convert_return(typ: BaseType, val: str) -> str:
         "_scaled_dot_product_flash_attention",
         "_scaled_dot_product_efficient_attention",
         "_scaled_dot_product_cudnn_attention",
+        "_scaled_dot_product_fused_attention_overrideable",
```

Comment:
Why add this here?

Reply:
Per my understanding, AOTI needs to check if an operation returns empty tensor potentially and then generate c_shim code accordingly. Currently, I think `_scaled_dot_product_fused_attention_overrideable` should have similar behaviors with other `_scaled_dot_product` operations, say it may return empty tensor.